browser/src/dashboard/Dashboard.tsx

```diff
@@ -313,6 +313,14 @@ export class Dashboard extends React.Component<DashboardProps, DashboardState> {
             </header>
           </div>
         </div>
+        <div className="govuk-grid-row govuk-body govuk-!-margin-bottom-4">
+          <div className="govuk-grid-column-full">
+            <div className="event-card" style={{ padding: '15px', paddingLeft: '20px', paddingRight: '20px' }}>
+              <h1 className="govuk-heading-l">Live payments dashboard</h1>
+              <p className="govuk-body-l" style={{ marginBottom: '20px' }}><a href="https://payments.service.gov.uk" className="govuk-link govuk-link--no-visited-state">GOV.UK Pay</a> is a free service, available to public sector organisations to take online payments.</p>
```

Comment:
We could reuse the ‘GOV.UK Pay is a secure online payment service for government and public sector organisations’ line from our support ticket macro.

Reply:
Ah I was going to say the above was from the product pages but it looks like we actually use both:

![Screenshot 2022-09-13 at 12 13 14](https://user-images.githubusercontent.com/2844572/189888842-6e3670bd-ce72-4f2e-8f64-ada067d8cdd8.png)

![Screenshot 2022-09-13 at 12 13 53](https://user-images.githubusercontent.com/2844572/189888865-68e605d7-3310-4a2c-928f-832d5406f26f.png)